build/.moduleignore

```diff
@@ -110,6 +110,9 @@ node-pty/third_party/**
 @parcel/watcher/src/**
 !@parcel/watcher/build/Release/*.node
 
+# exclude parcel watcher prebuilds
+@parcel/watcher-**
```

Comment:
given we remove them at install stage, is this needed ?

Reply:
Ok lets remove it.